opts/opts.go

```diff
@@ -177,7 +177,7 @@ type ValidatorFctType func(val string) (string, error)
 type ValidatorFctListType func(val string) ([]string, error)
 
 func ValidateLogOpts(val string) (string, error) {
-	allowedKeys := map[string]string{}
+	allowedKeys := map[string]string{"max-file": "1", "max-size": "2"}
```

Comment:
This validates log-opts for the daemon. The log drivers are called only while creating a container. A validation step is needed at this point.